util.c

```diff
@@ -109,15 +299,15 @@ check_user_token (const char *authfile,
 		  const char *username,
 		  const char *otp_id,
 		  int verbose,
-                  FILE *debug_file)
+      FILE *debug_file)
```

Comment:
Please avoid non-related changes like these (even if they make sense). If you want to make them create a separate PR with only that.

Reply:
Alright, taken into account